src/modelize/modelize_property.nit

```diff
@@ -1181,6 +1181,10 @@ redef class AAttrPropdef
 	# Could be through `n_expr`, `n_block` or `is_lazy`
 	var has_value = false
 
+	# The name of the attribute
+	# Note: The name of the attribute is in reality the name of the mreadpropdef
+	var name: String = n_id2.text is lazy
```

Comment:
the move of the property `name` should have been in the previous commit (instead of a remove then an add)